File: docs/src/content/code/python/validate_error.py
from lcax import ValidationSchema, validate, Level, ValidationRule, Project
from pathlib import Path

project = Project.loads(Path("project.json").read_text())

schemas = [
    ValidationSchema(**{
        "level": Level.Project,
        "field": "name",
        "rule": ValidationRule(**{"equal": "Te eksempel"})
    })
]

result = validate(project, schemas)
# result = [{ field: "name", message: "Field is not equal to: Te eksempel" }]

File: docs/src/content/code/javascript/validateError.ts
import {validate, ValidationRule, ValidationSchema} from "lcax";
import fs from 'fs'


const project_data =  fs.readFileSync('project.json')
const project = JSON.parse(project_data)
const schemas: ValidationSchema[] = [{
    level: 'project',
    field: 'name',
    rule: {equal: "Te eksempel"} as ValidationRule
}]

const result = validate(project, schemas)
// result = [{ field: "name", message: "Field is not equal to: Te eksempel" }]
